### DISCUSSION TITLE
Range of acceptable TestDataSet GM perfusion values

### DISCUSSION THREAD

**Discussion by schdempsey** [ExploreASL/ExploreASL#1433] [Q&A] [answered]
ExploreASL version 1.10.1

In the Quality Control tutorial, I've run the TestDataSet Quality=0 to test if I get the ~65ml/100g/min. I'm just confused about the word approximately here. The values I'm getting with 2023a are ~51. I've run the same in 2019a and returned the same. 

Running using version 1.10 in 2023a also reports ~51. the PVC2 file reports 65. Can anyone confirm 51/100g/min is fine to move ahead with figuring about compatibility with my own dataset?

**Comment by MDijsselhof** [comment, marked as answer]
Thank you for using ExploreASL! I apologise for the late reply. To answer your question: you are good to go! It appears we have made a mistake in the tutorial, thank you for pointing that out. At Quality=0 the segmentation algorithms are a little less accurate but far quicker. Corresponding values for GM CBF should be, without partial volume correction (PVC0) and with (PVC2) correspondingly ~51 and ~65.
I hope this helps!

**Reply by schdempsey** [reply]
Perfect!
